contrib/datawave-quickstart/bin/modification.sh

```diff
@@ -0,0 +1,295 @@
+   if [ "${DW_MODIFICATION_COMMAND}" == "INSERT" ] ; then
+       [ -z "${DW_MODIFICATION_NEW_VALUE}" ] && error "New field value is required" && return 1
+   elif [ "${DW_MODIFICATION_COMMAND}" == "REPLACE" ] ; then
+       [ -z "${DW_MODIFICATION_NEW_VALUE}" ] && error "New field value is required" && return 1
+   elif [ "${DW_MODIFICATION_COMMAND}" == "UPDATE" ] ; then
+       [ -z "${DW_MODIFICATION_NEW_VALUE}" ] && error "New field value is required" && return 1
+       [ -z "${DW_MODIFICATION_OLD_VALUE}" ] && error "Old field value is required" && return 1
+       BODY="<DefaultUUIDModificationRequest xmlns=\"http://webservice.datawave/v1\" _class=\"datawave.webservice.modification.DefaultUUIDModificationRequest\"><Events><Event><id>${DW_MODIFICATION_UUID}</id><idType>${DW_MODIFICATION_UUID_TYPE}</idType><operations><operation><operationMode>${DW_MODIFICATION_COMMAND}</operationMode><fieldName>${DW_MODIFICATION_FIELD}</fieldName><fieldValue>${DW_MODIFICATION_NEW_VALUE}</fieldValue><oldFieldValue>${DW_MODIFICATION_OLD_VALUE}</oldFieldValue><columnVisibility>$( xmlencode ${DW_MODIFICATION_VIZ} )</columnVisibility></operation></operations><user>testUser</user></Event></Events><mode>INSERT</mode><fieldName>TEST</fieldName><fieldValue>ABC</fieldValue><columnVisibility>PUBLIC</columnVisibility></DefaultUUIDModificationRequest>"
+   elif [ "${DW_MODIFICATION_COMMAND}" == "DELETE" ] ; then
+       [ -z "${DW_MODIFICATION_OLD_VALUE}" ] && error "Old field value is required" && return 1
+       BODY="<DefaultUUIDModificationRequest xmlns=\"http://webservice.datawave/v1\" _class=\"datawave.webservice.modification.DefaultUUIDModificationRequest\"><Events><Event><id>${DW_MODIFICATION_UUID}</id><idType>${DW_MODIFICATION_UUID_TYPE}</idType><operations><operation><operationMode>${DW_MODIFICATION_COMMAND}</operationMode><fieldName>${DW_MODIFICATION_FIELD}</fieldName><oldFieldValue>${DW_MODIFICATION_OLD_VALUE}</oldFieldValue><columnVisibility>$( xmlencode ${DW_MODIFICATION_VIZ} )</columnVisibility></operation></operations><user>testUser</user></Event></Events><mode>INSERT</mode><fieldName>TEST</fieldName><fieldValue>ABC</fieldValue><columnVisibility>PUBLIC</columnVisibility></DefaultUUIDModificationRequest>"
+   else
+       error "Command set to ${DW_MODIFICATION_COMMAND}.  Command must be one of INSERT, UPDATE, DELETE, or REPLACE." && return 1
+   fi
+
+   DW_CURL_DATA="-d '$BODY'"
+}
+
+function modificationHelp() {
+    echo
+    echo " The $( printGreen "datawaveModification" ) shell function allows you submit modification requestson demand to DataWave's"
```

Comment:
```suggestion
    echo " The $( printGreen "datawaveModification" ) shell function allows you submit modification requests on demand to DataWave's"
```
Minor user facing typo